src/main/resources/application.yaml

```diff
@@ -72,8 +72,8 @@ wellknown:
       fingerprint: "15:7E:42:A7:92:6B:A5:CA:9E:B0:29:8E:88:EE:81:0D:C3:13:E9:B5:84:41:50:28:8A:88:17:B4:14:40:FB:42"
 
 csp:
-  default-config: "default-src 'self';script-src 'self';style-src 'self';font-src 'self';img-src 'self';connect-src 'self';"
+  default-config: "default-src 'self';script-src 'self';style-src 'self' 'unsafe-inline';font-src 'self';img-src 'self';connect-src 'self';"
```

Comment:
why do we add `unsafe-line`?

Reply:
That was the quick (and a bit dirty) way to change the styling. Will add the inline styles into the css file, so that this can be removed.